models/migrations/v18.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2017 The Gogs Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import (
+	"fmt"
+
+	"github.com/go-xorm/xorm"
+	"github.com/gogs/gogs/pkg/setting"
```

Comment:
Prepend an empty line right before this line.

Reply:
Done